_overviews/contributors/index.md

```diff
@@ -0,0 +1,809 @@
+{% endhighlight %}
+
+The `ParadoxPlugin` is responsible of generating the website, and the `ParadoxSitePlugin` provides
+integration with `sbt-site`.
+The second line is optional, it defines the location of the website source files. In our case, in
+`src/documentation`.
+
+Add your documentation entry point in an `src/documentation/index.md` file. A typical documentation entry point
+uses the library name as title, shows a short sentence describing the purpose of the library, and a code
+snippet for adding the library to a build definition:
+
+{% highlight markdown %}
+# Library Example
+
+A library that does nothing.
+
+## Setup
+
+Add the following dependency to your `build.sbt` file:
+
+@@@vars
+~~~ scala
+libraryDependencies += "ch.epfl.scala" %% "library-example" % "$project.version$"
+~~~
+@@@
+
+@@@ index
+* [Getting Started](getting-started.md)
+* [Reference](reference.md)
+@@@
+{% endhighlight %}
+
+Note that in our case we rely on a variable substitution mechanism to inject the correct version number
+in the documentation so that we don’t have to always update that part of the docs each time we publish a
+new release.
+
+Our example also includes an `@@@index` directive, defining how the content of the documentation is organized.
+In our case, the documentation contains two pages, the first one provides a quick tutorial for getting
+familiar with the library, and the second one provides more detailed information.
+
+The sbt-site plugin provides a convenient `previewAuto` task that serves the resulting documentation locally,
+so that you can see how it looks like, and re-generate the documentation when you edit it:
+
+~~~
+sbt:library-example> previewAuto
+Embedded server listening at
+  http://127.0.0.1:4000
+Press any key to stop.
+~~~
+
+Browse the [http://localhost:4000](http://localhost:4000) URL to see the result:
+
+![](/resources/images/documentation-preview.png)
+
+### Include Code Examples
+
+This section shows two ways to make sure that code examples included in the documentation do compile
+and behave as they are presented.
+
+#### Using a Markdown Preprocessor
+
+One approach consists in using a Markdown preprocessor, such as [tut](https://github.com/tpolecat/tut) or
+[mdoc](https://github.com/olafurpg/mdoc). These tools read your Markdown source files, search for code fences,
+evaluate them (throwing an error if they don’t compile), and produce a copy of your Markdown files where
+code fences have been updated to also include the result of evaluating the Scala expressions.
+
+For instance, given the following `src/documentation/getting-started.md` file:
+
+{% highlight markdown %}
+# Getting Started
+
+First, start with the following import:
+
+```tut
+import ch.epfl.scala.Example
+```
+
+Then, do nothing with something:
+
+```tut
+Example.doNothing(42)
+```
+{% endhighlight %}
+
+The tut tool will produce the following Markdown file:
+
+{% highlight markdown %}
```

Comment:
The CI fails because my Markdown file contains a Markdown snippet that contains a `tut` snippet. I’m not sure how I can write my Markdown snippet in such a way that `tut` ignores its content.